packages/meteor/dynamics_browser.js

```diff
@@ -28,6 +28,17 @@ EVp.withValue = function (value, func) {
   return ret;
 };
 
+EVp.set = function (context) {
+  currentValues[this.slot] = context;
+};
+
+EVp.setNewContextAndGetCurrent = function (value) {
+  const saved = currentValues[this.slot];
+  this.set(value);
+  return saved;
+};
```

Comment:
1. With my two comments, these two are most likely obsolete.
2. If they're used only internally, maybe these should be private? I'm still _a little_ against increasing the API surface if it's not needed.

Reply:
We still need these, but I made them private.